tgui/packages/tgui/interfaces/AntagInfoGeneric.tsx

```diff
@@ -28,3 +34,277 @@ export const AntagInfoGeneric = (props, context) => {
     </Window>
   );
 };
+
+// SKYRAT EDIT ADDITION START
+const Rules = (props, context) => {
+  const { data } = useBackend<Info>(context);
+  const { antag_name } = data;
+  switch (antag_name) {
+    case 'Abductor Agent' || 'Abductor Scientist' || 'Abductor Solo':
+      return (
+        <Stack vertical>
+          <Stack.Item bold>Special Rules:</Stack.Item>
+          <Stack.Item>
+            {
+              '- Essentially, picture Destroy All Humans(game series) without the personality, and you have the right idea.'
+            }
+          </Stack.Item>
+          <Stack.Item>
+            {
+              '- Functionally PMS once called out, but you should avoid just running around doing nothing but mechanics if possible.'
+            }
+          </Stack.Item>
+          <Stack.Item>
+            {
+              "- You're very-PMS if you're openly using your gear, this includes the tools or elsewise, so no, the crew do not have to wait for you to hack the door open before running away."
+            }
+          </Stack.Item>
+          <Stack.Item bold>Metaprotections:</Stack.Item>
+          <Stack.Item>
+            {
+              '- The Science department knows that Grays consistently steal away people for strange experiments. They are familiar with their ability to teleport on-station, as well as disguise themselves. They know that people abruptly appearing detained in strange places can be a sure sign of a freshly experimented-on crewmember. Especially if their memory is gone, or they’re acting strange. They also know they can’t speak vocally.'
+            }
+          </Stack.Item>
+          <Stack.Item>
+            {
+              "- The Medical department: is aware of Abductors, and their habit for replacing organs, once they're confirmed, medical should be on the prowl for people behaving weirdly, or else-wise, vanishing for periods of time--essentially they know everything science knows."
+            }
+          </Stack.Item>
+          <Stack.Item>
+            {
+              '- The Security department is aware of an alien species prone to abducting crew, and that you have to go above-and-beyond with them to put them down, due to their advanced armors.'
+            }
+          </Stack.Item>
+          <Stack.Item>
+            {
+              "- Everyone knows that abductors are bad-news-bears, have a bad habit of appearing where you don't want to be, and elsewise, know it's a good habit to disable your cameras if they're about, because they have a funny habit of tapping into them."
```

Comment:
```suggestion
              "- Everyone knows that Abductors are bad-news-bearers, have a bad habit of appearing where you don't want to be, and elsewise, know it's a good habit to disable your cameras if they're about, because they have a funny habit of tapping into them."
```